rclpy/package.xml

```diff
@@ -34,8 +34,10 @@
   <depend>rosidl_runtime_c</depend>
   <depend>unique_identifier_msgs</depend>
 
+  <exec_depend>action_msgs</exec_depend>
   <exec_depend>ament_index_python</exec_depend>
   <exec_depend>builtin_interfaces</exec_depend>
+  <exec_depend>lifecycle_msgs</exec_depend>
```

Comment:
Actually, `lifecycle_msgs` should be a `<depend>`; it is required for building the C++ side of things as well.

Reply:
Nice catch [37333a3](https://github.com/ros2/rclpy/pull/1103/commits/37333a34bc51482f07b907e51bfd06f2b6e74192)